etc/restic/default.env.sh

```diff
@@ -32,6 +32,7 @@ export RESTIC_BACKUP_TAG=systemd.timer
 
 # Retention policy - How many backups to keep.
 # See https://restic.readthedocs.io/en/stable/060_forget.html?highlight=month#removing-snapshots-according-to-a-policy
+export RESTIC_RETENTION_HOURS=1
```

Comment:
You probably want to keep at least 24 or 36 hours, to span a bit over daily snapshots.

Reply:
Good point about not changing existing behavior too much.

Note: making use of hourly snapshots will require to also alter the systemd timer or cron rules.